packages/bootstrap/scss/skeleton/_variables.scss

```diff
@@ -1,11 +1,24 @@
 // Skeleton
+
+/// The transform scale of the Skeleton text.
+/// @group skeleton
 $kendo-skeleton-text-transform: scale( 1, .6 ) !default;
+/// The border radius of the Skeleton text.
+/// @group skeleton
 $kendo-skeleton-text-border-radius: $kendo-border-radius-md !default;
 
-$kendo-skeleton-rect-border-radius: 0 !default;
 
+/// The border radius of the rectangle Skeleton.
+/// @group skeleton
+$kendo-skeleton-rect-border-radius: 0 !default;
+/// The border radius of the circle Skeleton.
```

Comment:
Same as above, circular instead of circle?

Reply:
This applies to the other themes as well, also maybe we should separate the two variables with a newline for consistency?